tests/e2e/utils/clear-cookies.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * `clearCookiesByPrefix` utility.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Clears cookies by prefix.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Array.<string>} cookiePrefixes Cookie prefixes.
+ */
+export async function clearCookiesByPrefix( ...cookiePrefixes ) {
+	await page.evaluate( ( prefixes ) => {
+		const cookieNames = document.cookie.split( /=[^;]*(?:;\s*|$)/ );
```

Comment:
There's a cleaner way to do this using Puppeteer's API – see https://github.com/google/site-kit-wp/blob/3b92666185596772a90f0b1f22537fabd4b85b14/tests/e2e/utils/delete-auth-cookie.js

That utility could also be rewritten to use this one internally but we don't need to do that here.

Reply:
Thanks for pointing that out! I had taken a quick look but somehow missed that, and already had a rough version of the cookie clearing code as a snippet from my testing, so just included that. I've now removed it and used the Puppeteer API instead.